src/containers/BooksList.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Book from '../components/Book';
+import { removeBook, changeFilter } from '../actions/index';
+import CategoryFilter from '../components/CategoryFilter';
+
+class BooksList extends Component {
+  constructor(props) {
+    super(props);
+    this.removeBookFromState = this.removeBookFromState.bind(this);
+    this.filterBooks = this.filterBooks.bind(this);
+  }
+
+  filterBooks({ target: { value: filter } }) {
```

Comment:
- [ ] [OPTIONAL] Fun fact: As mentioned at the milestone 3 review, if you make this function using arrow functions you won't need to bind it inside the constructor.